src/main/resources/public/js/wallets.js

```diff
@@ -380,14 +378,14 @@ function payInvoice(invoice, contract, payButton) {
                             );
                     }
                 );
-                if(jqXHR.status == 412) {
-                    alert(JSON.parse(jqXHR.responseText).message);
-                } else {
-                    alert(
-                        "Something went wrong (" + jqXHR.status + ")." +
-                        "Please, refresh the page and try again."
-                    );
-                }
+                var message = (jqXHR.status == 412)
+                    ? JSON.parse(jqXHR.responseText).message
+                    : "Something went wrong (" + jqXHR.status + "). Please try again.";
+                confirmDialog
+                    .create(message, "Error", "Yes")
+                    .then(()=> {
+                        payInvoice(invoice, contract, payButton);
+                    });
```

Comment:
@criske I think this change is for ticket https://github.com/self-xdsd/self-web/issues/428, right? If yes, please don't display the status. Just "Something went wrong. Please try again."

Reply:
@amihaiemil This is has the original logic, but it uses  a  bootstrap dialog instead of alert. #428 is asking for a 500 check.
before:
```javascript
if(jqXHR.status == 412) {
   alert(JSON.parse(jqXHR.responseText).message);
 } else {
   alert("Something went wrong (" + jqXHR.status + ")." + "Please, refresh the page and try again.");
}
```

Reply:
@criske right, my bad